File: Arrays/Largest_Element/sol.py
def findLargestElement(arr, n):
    max = arr[0]
    for i in range(1,n):
        if arr[i] > max:
            max = arr[i]

    return max


if __name__ == "__main__":

    arr1 = [2, 5, 1, 3, 0]
    n = len(arr1)  # Size of the array
    max = findLargestElement(arr1, n)  # Call the function to find the largest element
    print("The largest element in the array is:", max)  # Output the result

    

File: Arrays/Largest_Element/sol.cpp
#include <bits/stdc++.h>
using namespace std;

int findLargestElement(int arr[], int n){
    int max = arr[0];
    for (int i = 0; i< n; i++){
        if (arr[i] > max){
            max = arr[i];
        }
    }
    return max;
}


int main(){
    int arr[] = {1, 2, 3, 4, 5};
    int n = sizeof(arr)/sizeof(arr[0]);
    cout << "Largest Element is: " << findLargestElement(arr, n) << endl;
    return 0;
}
